onnxruntime/core/framework/sequential_executor.cc

```diff
@@ -306,7 +307,7 @@
 #endif
 
 #ifdef DEBUG_NODE_INPUTS_OUTPUTS
-    session_scope_.dump_context_.program_counter = program_counter_++;
+    session_scope_.dump_context_.program_counter = session_scope.program_counter_++;
```

Comment:
>session_scope_.dump_context_.program_counter  [](http://example.com/codeflow?start=4&length=45)

Pls be mindful of the approach that we are using a session_state::dump_context for dum, in multi-stream case, there might be some race condition when consuming the ::program_counter